### PR TITLE
Add color prop to TextComponent

### PR DESCRIPTION
**Pull request by patrickkim**

### CODE REVIEW

File: src/atoms/Color/text_color.module.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@each $key, $value in $colors-map {
```
Comment (jmcolella) [inline src/atoms/Color/text_color.module.scss]: why did you add this file? just curious

**Comment by drewdrewthis** [comment]
@patrickkim Before you merge

File: src/atoms/Color/text_color.module.scss (hunk 1 above)
Comment (patrickkim) [inline src/atoms/Color/text_color.module.scss]: my mistake. got in there by accident.

**Comment by patrickkim** [comment]
I was deferring that decision for now. I added a chore to handle this in a cleaner way on the css side.
